src/main/java/com/minecolonies/coremod/colony/jobs/AbstractJob.java

```diff
@@ -73,6 +67,25 @@ public AbstractJob(final CitizenData entity)
     }
 
     /**
+     * Suppressing Sonar Rule squid:S2390
+     * This rule does "Classes should not access static members of their own subclasses during initialization"
+     * But in this case the rule does not apply because
+     * We are only mapping classes and that is reasonable
+     */
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this variable to comply with Java Code Conventions. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1213)